frontend/ui/src/pages/Inbox/Messenger/MessageList/index.tsx

```diff
@@ -165,12 +168,18 @@ const MessageList = (props: MessageListProps) => {
                   {formatDateOfMessage(message)}
                 </div>
               )}
-              <SourceMessage
-                source={getSource(conversation)}
-                message={message}
-                contact={contactToShow}
-                lastInGroup={lastInGroup}
-              />
+              <MessageInfoWrapper
+                fromContact={isFromContact(message)}
+                contact={conversation.metadata.contact}
+                sentAt={sentAt}
+                lastInGroup={lastInGroup}>
+                <SourceMessage
+                  source={getSource(conversation)}
+                  message={message}
+                  contact={contactToShow}
```

Comment:
Is the `contact` still needed for the `SourceMessage` after this PR? If not I would remove it here and from the `DefaultMessageRenderingProps` :)